posthog/api/test/test_insight.py

```diff
@@ -759,7 +759,8 @@ def test_serializer(self, patch_capture_exception):
             f"/api/projects/{self.team.id}/insights/funnel/",
             {"events": events, "breakdown": [123, 8124], "breakdown_type": "cohort"},
         )
-        # self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.status_code, 500)
```

Comment:
I don't have context on this test case (looks like it was added in https://github.com/PostHog/posthog/pull/8491), but I'm _pretty_ sure we should never assert a server error. Most likely this should be fixed. 🤔 Nagging the author @timgl to look at this

Reply:
Yeah, I couldn't deduce the correct test outcome :)

_maybe_ we need to create the cohorts so that the SQL query doesn't error 🤔